tests/difference/core/driver/trace.go

```diff
@@ -31,21 +31,21 @@ type Consequence struct {
 	T                   int      `json:"t,omitempty"`
 }
 
-type ActionAndConsequence struct {
-	Action      Action      `json:"action"`
-	Consequence Consequence `json:"consequence"`
-	Ix          int         `json:"ix"`
+type ActionAndPartialState struct {
+	Action       Action       `json:"action"`
+	PartialState PartialState `json:"partialState"`
+	Ix           int          `json:"ix"`
```

Comment:
More understandable:
```suggestion
	Index           int          `json:"index"`
```